tools/postprocess.py

```diff
@@ -141,13 +141,15 @@ def on_post_build(env):
                             + f"<h1{match.group(1)}>{match.group(2)}</h1>"
                             + html_content[match.end():])
                         file_was_changed = True
+                    #"""
                     # Collapse doubled <h1>/<h2> page titles
                     REPEATED_H1_H2 = re.compile(
                         r"<h1>(.*?)</h1>\s*<h2\s+(id=\".*?\")>\1(<a\s+class=\"headerlink\".*?</a>)?</h2>", re.M | re.S
                     )
                     html_content, n_changes = REPEATED_H1_H2.subn('<h1 \\2>\\1\\3</h1>', html_content)
                     if n_changes != 0:
                         file_was_changed = True
+                    #"""
```

Comment:
?

Reply:
Just a big fat comment to keep this code safe.